### PR TITLE
on boarding issue suggestions

### PR DESCRIPTION
**Pull request by ateucher**

### CODE REVIEW

**Comment by ateucher** [comment]
I don't know what I did here. Sorry about the duplicated commits!

**Comment by ateucher** [comment]
Other comments:
- I think it's good practice to add minimum version numbers to dependencies in Imports (I usually just use the current minor version on CRAN)
-

File: paper/onboarding_issue.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Anyone who is interested in making use of the Water Survey of Canada data source
...
 yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
 
-    - A search of the term "hydat" on github reveals two R packages that are provide similar (but not all the same) functionality:
-    - [HYDAT](https://github.com/CentreForHydrology/HYDAT): the original author is a contributor to this package and will not be further developing this package soon. 
-    -[hydatr](https://github.com/paleolimbot/hydatr) was develop at approximately the same time as `tidyhdyat`. 
+    - A search of the term "hydat" on github reveals two R packages that provide similar (but not all the same) functionality:
+    - [HYDAT](https://github.com/CentreForHydrology/HYDAT): the original author of **HYDAT** is a contributor to this package and will soon cease further development of **HYDAT**. 
+    -[hydatr](https://github.com/paleolimbot/hydatr) was developed at approximately the same time as `tidyhdyat`. 
     - `tidyhydat` documentation is more thorough than both packages.
     - Through the realtime functions `tidyhydat` provides greater functionality than both packages (webservice for both; datamart for `hydatr`)
```
Comment (ateucher) [inline paper/onboarding_issue.md]: I'm a bit unclear what you mean here (webservice for both; datamart for `hydatr`)

File: paper/onboarding_issue.md [hunk 2]
```diff
@@ -45,11 +45,11 @@ VignetteBuilder: knitr
...
+  - data extraction: The majority of `tidyhydat`'s exported functions provides easy access to Environment and Climate Change Canada's (ECCC) publicly available HYDAT sqlite3 database (http://collaboration.cmc.ec.gc.ca/cmc/hydrometrics/www/) of historical river and lake Canada. HYDAT is updated quarterly and is distributed via the [Canadian Open Government License](https://github.com/bcgov/tidyhydat/blob/master/data-raw/HYDAT_internal_data/LICENSE.OGL-CAN-2.0). HYDAT is not included with this package because it is regularly updated and because the file size is prohibitively large (>1GB) for inclusion in an R package. Rather `tidyhydat` provides the `download_hydat()` function that downloads it for the user. `download_realtime_ws` provides access via the `httr` package to a new password protected web service provided by ECCC. Differences between the download realtime functions are outlined [here](https://github.com/bcgov/tidyhydat/blob/master/vignettes/tidyhydat.Rmd#compare-download_realtime_ws-and-download_realtime_dd)
```
Comment (ateucher) [inline paper/onboarding_issue.md]: I think the discussion of `download_realtime_ws` belongs in the data retrieval vs data extraction section

Comment (boshek) [reply paper/onboarding_issue.md]: Actually @ateucher  I'm not quite sure what you mean here. Can you elaborate?

Comment (boshek) [reply paper/onboarding_issue.md]: Oh nevermind I think I've got it.

**Pull request merged**